libsplinter/src/biome/rest_api/mod.rs

```diff
@@ -117,7 +117,7 @@ use crate::rest_api::sessions::AccessTokenIssuer;
 /// * `GET /biome/user/{id}` - Retrieve user with specified ID
 /// * `DELETE /biome/user/{id}` - Remove user with specified ID
 pub struct BiomeRestResourceManager {
-    #[cfg(any(feature = "biome-key-management", feature = "biome-credentials",))]
+    #[cfg(feature = "biome-credentials")]
```

Comment:
These guards would presumably still be necessary when we have the feature turned off?

Reply:
Based on the way the routes are set up, the user store only appears to be required by the routes that are enabled by the `biome-credentials` feature

Reply:
Ok, please move that fix to another PR.